src/main/java/gregtech/api/fluids/fluidType/FluidType.java

```diff
@@ -0,0 +1,57 @@
+package gregtech.api.fluids.fluidType;
+
+import gregtech.api.unification.material.Material;
+import net.minecraftforge.fluids.Fluid;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.HashMap;
+import java.util.Map;
+
+public abstract class FluidType {
```

Comment:
Make this a CT class so you wouldn't have to manually resolve strings and even open up to adding new fluid types (?)